Sources/ArcGISToolkit/Resources/FloorFilter.storyboard

```diff
@@ -0,0 +1,254 @@
+                                        <rect key="frame" x="0.0" y="0.0" width="48" height="48"/>
+                                        <color key="backgroundColor" systemColor="systemGray5Color"/>
+                                        <constraints>
+                                            <constraint firstAttribute="width" constant="48" id="LuJ-hY-JGn"/>
+                                            <constraint firstAttribute="height" constant="48" id="NBH-4z-icg"/>
+                                        </constraints>
+                                        <color key="tintColor" white="0.33333333333333331" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                        <state key="normal" image="Close">
+                                            <preferredSymbolConfiguration key="preferredSymbolConfiguration" scale="small"/>
+                                        </state>
+                                    </button>
+                                    <tableView clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="none" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="28" sectionFooterHeight="28" translatesAutoresizingMaskIntoConstraints="NO" id="1OE-0w-M3G" userLabel="levelsList">
+                                        <rect key="frame" x="0.0" y="49" width="48" height="180"/>
+                                        <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                        <color key="tintColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                        <constraints>
+                                            <constraint firstAttribute="width" constant="48" id="mpJ-br-OIf"/>
+                                            <constraint firstAttribute="height" constant="180" id="zOh-xT-z68"/>
+                                        </constraints>
+                                        <color key="separatorColor" white="0.33333333329999998" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                        <prototypes>
+                                            <tableViewCell clipsSubviews="YES" contentMode="center" restorationIdentifier="levelCell" insetsLayoutMarginsFromSafeArea="NO" selectionStyle="blue" hidesAccessoryWhenEditing="NO" indentationWidth="0.0" shouldIndentWhileEditing="NO" reuseIdentifier="levelCell" rowHeight="40" id="E77-hT-5hd" userLabel="levelCell">
```

Comment:
Is this just a basic cell? It appears to be, since `UITableViewCell.textLabel` is being used. If so, the type should be `Basic`.

![image](https://user-images.githubusercontent.com/2257493/151226827-ea76316a-465a-4cfd-852d-14a9cf5a47c6.png)